tests/devices/default_qubit/test_default_qubit.py

```diff
@@ -27,6 +27,13 @@
 np.random.seed(0)
 
 
+max_workers_list = [
+    None,
+    pytest.param(1, marks=pytest.mark.slow),
+    pytest.param(2, marks=pytest.mark.slow),
```

Comment:
# Praise
Nice, I didn't know you could apply markers on a per-parameter basis! 🎉